examples/webgl_panorama_shaderMaterial.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>three.js webgl - panorama</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link type="text/css" rel="stylesheet" href="main.css">
+</head>
+
+<body>
+    <div id="container"></div>
+    <div id="info">
+        <a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - shaderMaterial panorama demo By <a href="https://github.com/hopepdm" target="_blank" rel="noopener">Hopepdm</a>
+    </div>
+
+    <script type="module">
+        import * as THREE from '../build/three.module.js';
+        import { OrbitControls } from './jsm/controls/OrbitControls.js';
+        import { GUI } from './jsm/libs/dat.gui.module.js';
+        
+        var camera, controls; 
+        var renderer;
+        var scene;
+        var uniforms;
+
+        init();
+        animate();
+
+        function init() {
+
+            var container = document.getElementById( 'container' );
+
+            renderer = new THREE.WebGLRenderer();
+            renderer.setPixelRatio( window.devicePixelRatio );
+            renderer.setSize( window.innerWidth, window.innerHeight );
+            container.appendChild( renderer.domElement );
+
+            scene = new THREE.Scene();
+
+            camera = new THREE.PerspectiveCamera( 90, window.innerWidth / window.innerHeight, 0.1, 1000 );
+            camera.position.z = 0.01;
+
+            controls = new OrbitControls( camera, renderer.domElement );
+            controls.enableZoom = false;
+            controls.enablePan = false;
+            controls.enableDamping = true;
+            controls.rotateSpeed = - 0.25;
+
+            var params = {
+                alp: 0,
+                opacity: 1
+            }
+
+            uniforms = {
+                map1: {
+                    type: 't',
```

Comment:
FYI - This `type` property is no longer needed.

Reply:
thank you for reminding me